docs/developer_guide/build.md

```diff
@@ -33,7 +33,7 @@ Read the [environment setup guide](developer_guide/env_setup.md) first to setup
 ## Build YuniKorn for Kubernetes
 
 Prerequisite:
-- Go 1.16+
+- Golang, check [yunikorn-k8shim/.go_version](https://github.com/apache/yunikorn-k8shim/blob/master/.go_version) to see the version of Yunikorn request.
```

Comment:
A bit more generic as we have multiple repos and all will have the same requirements not just the k8shim:
```
Golang: check the `.go_version` file in the root of the repositories for the version Yunikorn requires. The minimum version can change per release branch.  Earlier Go versions might cause compilation issues. 
```

Reply:
I am in favor of your suggestion, and update the content in the new commit. 
![{6F2EE5B5-2720-411B-8BD6-3FE8C5AB14B0}](https://user-images.githubusercontent.com/48400525/203690724-a2678c8b-5b42-47a1-a8d5-6a73c0aa59d6.png)